packages/compiler-vapor/src/transform.ts

```diff
@@ -444,6 +444,20 @@ function transformProp(
       })
       break
     }
+    case 'show': {
+      if (expr === null) {
+        // TODO: support v-show
+        return
+      }
+
+      ctx.registerEffect(expr, {
```

Comment:
Does this implementation require further design and discussion @sxzz? Currently, vapor lacks some runtime design. If it needs to be functionally unified with vue/core, then it needs a life cycle design, such as the life cycle of instructions and the life cycle of components. In vue/core v-model and v-show rely on the life cycle of instructions for implementation. I am not sure whether this is needed in vapor.

Reply:
Emm I will make some changes according to the design after @sxzz to reply.

Reply:
Sure, we need to implement basic runtime directives first
See https://vuejs.org/guide/reusability/custom-directives.html#directive-hooks